src/foam/u2/view/UnstyledTableView.js

```diff
@@ -319,7 +311,12 @@ foam.CLASS({
 
     async function initE() {
       var view = this;
-
+      const asyncRes = await this.filterUnpermited(view.of.getAxiomsByClass(foam.core.Property));
+      this.allColumns = ! view.of ? [] : [].concat(
```

Comment:
please add  
`const asyncProps = await this.filterUnpermited(this.props.map(p => p.property));
this.props = asyncProps;
`